packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -397,6 +397,8 @@ const fixedNodeType = ({ name, getTracedSVG }) => {
     },
     resolve: (image, options, context) =>
       Promise.resolve(resolveFixed(image, options)).then(node => {
```

Comment:
I'm looking at this and I'm convinced it could just be

```
resolve: (image, options, context) =>
  Promise.resolve(resolveFixed(image, options)).then(node => {
```
into
```
resolve: async (image, options, context) =>
  const node = resolveFixed(image, options)
```

And it would be the same thing. The `resolveFixed` func is never returning a promise so there's no need (anymore?) for the `Promise.resolve`. The `async` keyword will ensure a promise is returned to preserve that api.

Not sure if it makes any difference but would be clearer. Oh well, this was unrelated to this PR, I was just confused looking at it :)